R/parameters.R

```diff
@@ -0,0 +1,7 @@
+# Allowed parameter ids which we can pass to the model
+expected_parameters <- c(
+  "country",
+  "pathogen",
+  "response",
+  "vaccine"
+)
```

Comment:
Maybe a bit misleading to store data that is used only by a test, outside of the tests folder.

Reply:
I’ll let @pratikunterwegs opine.

Reply:
I think it could live in tests for now and move to internal package data once the need becomes clear.